html/pfappserver/lib/pfappserver/Model/Interface.pm

```diff
@@ -403,6 +403,7 @@ sub getType {
         else {
             $type = $interface->{type};
             $type = ($type =~ /management|managed/i) ? 'management' : 'other';
+            $type .= ($interface->{type} =~ /portal/i) ? ',portal' : '';
```

Comment:
Actually no... since it mess up with something else...
I fixed it in the template for the moment... again, it NEEDS polishing... will be some other work